.github/workflows/combined-workflow-with-docs.yaml

```
@@ -0,0 +1,55 @@
+    uses: ./.github/workflows/deploy-pages.yaml
+    with:
+      coverage_artifact: coverage-reports
+      api_docs_artifact: api-docs
+
+  call-php-build-images:
+    needs: [call-php-code-quality, call-php-unit-tests]
+    if: |
+      always() &&
+      needs.call-php-code-quality.result == 'success' &&
+      needs.call-php-unit-tests.result == 'success'
+    uses: ./.github/workflows/php-build-images.yaml
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)